content/api/downtimes/downtimes_schedule.md

```diff
@@ -1,31 +1,30 @@
 ---
 title: Schedule monitor downtime
 type: apicontent
-order: 8.1
+order: 9.1
 external_redirect: /api/#schedule-monitor-downtime
 ---
 
 ## Schedule monitor downtime
 
 ##### ARGUMENTS
 
-
-* **`scope`** [*required*]:  
+*   **`scope`** [*required*]:
```

Comment:
This file has new indentations that aren't part of the monitor lists PR. Is this necessary?